src/core/jvm/insn/VariableInstruction.js

```diff
@@ -0,0 +1,25 @@
+import * as _ from 'lodash';
+
+const AbstractInstruction = require('./AbstractInstruction');
+const ImmediateByteInstruction = require('./ImmediateByteInstruction');
+
+export default class VariableInstruction extends AbstractInstruction {
+
+  constructor(instruction) {
+    super(instruction.idx, instruction.opcode);
+    this.instruction = instruction;
+  }
+
+  get var() {
+    if (this.instruction.constructor.name === 'ImmediateByteInstruction') {
+      return this.instruction.val;
+    } else {
+      if (this.instruction.opname.includes('M1')) {
```

Comment:
Let's move this up to make it `} else if (this.instruction...` so we don't have deep nesting.